layouts/v2-sidebar.html

```diff
@@ -17,6 +17,7 @@ <h4 class="sidebar-title">API v2</h4>
   <li><ul class="nav-list">
     <li><a href="/v2/domains/certificates/">Certificates</a></li>
     <li><a href="/v2/domains/email-forwards/">Email Forwards</a></li>
+    <li><a href="/v2/domains/templates/">Templates</a></li>
```

Comment:
@jacegu You're right. I moved them under Templates section. The structure now reflects Services > Domains.